spec/spec.md

```diff
@@ -416,6 +427,7 @@ where:
             "display_address_format": "<display_address_format>",
             “label”: “<label>”, 
             "icon": "<icon>",
+            "profile": "<account_profile_id>",
```

Comment:
caveat here: adding this parameter to the account struct deviates from the wallet-standard [Wallet Account API](). To support this on the web we will need to map this profile id into the `features` parameter. 

Alternatively, we could do the opposite. We could drop this new parameter in the spec, and instead use the existing `features` parameter. In the mobile SDKs, we could abstract this into an explicit `profile` parameter.

Reply:
@sdlaver

Reply:
I'd put it into the `features` parameter.